ModsenOnlineStore.EmailAuthentication.Infrastructure/Services/EmailSendingService.cs

```diff
@@ -0,0 +1,39 @@
+﻿using ModsenOnlineStore.EmailAuthentication.Application.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Mail;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ModsenOnlineStore.EmailAuthentication.Infrastructure.Services
+{
+    public class EmailSendingService : IEmailSendingService
+    {
+        private MailAddress from = new MailAddress("akkdliabreda@gmail.com", "Store Mail Authentication");
+        private NetworkCredential networkCredential = new NetworkCredential("akkdliabreda@gmail.com", "vqshkhtemsppmhha");
+
+        public void SendAuthEmail(string mailAddress = "egrom2002@gmail.com", string title = "", string text = "")
```

Comment:
It is better to name just SendEmail as nothing here about auth

Reply:
fixed